src/test/java/byteceps/storage/StorageTest.java

```diff
@@ -204,7 +205,7 @@ public void execute_load_corruptedJSONFailure() {
 
         assertDoesNotThrow( () -> checkFile(corruptFile));
 
-        Storage failureStorage = new Storage(corruptFile);
+        Storage failureStorage = new Storage(corruptFile, new UserInterface());
```

Comment:
Are we able to use the storage instance that we have in the setup() method?

Reply:
The storage instance in setup() uses data.json by default. This failure test case needs to corrupted.json for it to function. Apologies